### ISSUE TITLE
Internal: failed call to cuDevicePrimaryCtxRetain: CUDA_ERROR_LAUNCH_FAILED

### ISSUE THREAD

**Issue by Phill-at-GE** [tensorflow/tensorflow#51400]
<em>Please make sure that this is a bug. As per our
[GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md),
we only address code/doc bugs, performance issues, feature requests and
build/installation issues on GitHub. tag:bug_template</em>

**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): No
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Linux Ubuntu 20.04
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device: N/A
- TensorFlow installed from (source or binary): Conda
- TensorFlow version (use command below): 2.4.1
- Python version: 3.8.8
- Bazel version (if compiling from source): N/A
- GCC/Compiler version (if compiling from source): N/A
- CUDA/cuDNN version: 10.1.168/7.6.5
- GPU model and memory: GeForce GTX 1080 Ti computeCapability: 6.1
coreClock: 1.582GHz coreCount: 28 deviceMemorySize: 10.92GiB deviceMemoryBandwidth: 451.17GiB/s

You can collect some of this information using our environment capture
[script](https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh)
You can also obtain the TensorFlow version with:
1. TF 1.0: `python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"`
2. TF 2.0: `python -c "import tensorflow as tf; print(tf.version.GIT_VERSION, tf.version.VERSION)"`

**Describe the current behavior**
Can't complete any TensorFlow Operations (i.e. import tensorflow as tf;print(tf.reduce_sum(tf.random.normal([1000, 1000]))))

**Describe the expected behavior**
Complete TensorFlow Operations

**[Contributing](https://www.tensorflow.org/community/contribute)**

- Do you want to contribute a PR? (yes/no): No
- Briefly describe your candidate solution(if contributing):

**Standalone code to reproduce the issue**
Provide a reproducible test case that is the bare minimum necessary to generate
the problem. If possible, please share a link to Colab/Jupyter/any notebook.
```
import os
os.environ['TF_XLA_FLAGS'] = '--tf_xla_enable_xla_devices'
import tensorflow as tf;print(tf.reduce_sum(tf.random.normal([1000, 1000])))
```

**Other info / logs** Include any logs or source code that would be helpful to
diagnose the problem. If including tracebacks, please include the full
traceback. Large logs and files should be attached.
Logs:
```
(base) user@computer:~$ python
Python 3.8.8 (default, Apr 13 2021, 19:58:26) 
[GCC 7.3.0] :: Anaconda, Inc. on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import os
>>> os.environ['TF_XLA_FLAGS'] = '--tf_xla_enable_xla_devices'
>>> import tensorflow as tf;print(tf.reduce_sum(tf.random.normal([1000, 1000])))
2021-08-09 18:25:50.895134: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcudart.so.10.1
2021-08-09 18:25:51.967116: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcuda.so.1
2021-08-09 18:25:52.035250: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1720] Found device 0 with properties: 
pciBusID: 0000:05:00.0 name: GeForce GTX 1080 Ti computeCapability: 6.1
coreClock: 1.582GHz coreCount: 28 deviceMemorySize: 10.92GiB deviceMemoryBandwidth: 451.17GiB/s
2021-08-09 18:25:52.036032: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1720] Found device 1 with properties: 
pciBusID: 0000:06:00.0 name: GeForce GTX 1080 Ti computeCapability: 6.1
coreClock: 1.582GHz coreCount: 28 deviceMemorySize: 10.92GiB deviceMemoryBandwidth: 451.17GiB/s
2021-08-09 18:25:52.037043: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1720] Found device 2 with properties: 
pciBusID: 0000:09:00.0 name: GeForce GTX 1080 Ti computeCapability: 6.1
coreClock: 1.582GHz coreCount: 28 deviceMemorySize: 10.92GiB deviceMemoryBandwidth: 451.17GiB/s
2021-08-09 18:25:52.037996: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1720] Found device 3 with properties: 
pciBusID: 0000:0a:00.0 name: GeForce GTX 1080 Ti computeCapability: 6.1
coreClock: 1.582GHz coreCount: 28 deviceMemorySize: 10.92GiB deviceMemoryBandwidth: 451.17GiB/s
2021-08-09 18:25:52.038016: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcudart.so.10.1
2021-08-09 18:25:52.039398: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcublas.so.10
2021-08-09 18:25:52.039431: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcublasLt.so.10
2021-08-09 18:25:52.040687: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcufft.so.10
2021-08-09 18:25:52.040882: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcurand.so.10
2021-08-09 18:25:52.042260: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcusolver.so.10
2021-08-09 18:25:52.043012: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcusparse.so.10
2021-08-09 18:25:52.045963: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcudnn.so.7
2021-08-09 18:25:52.052758: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1862] Adding visible gpu devices: 0, 1, 2, 3
2021-08-09 18:25:52.053048: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  SSE4.1 SSE4.2 AVX AVX2 FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2021-08-09 18:25:52.072766: I tensorflow/core/platform/profile_utils/cpu_utils.cc:112] CPU Frequency: 3597710000 Hz
2021-08-09 18:25:52.073339: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x5618cc45fbc0 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2021-08-09 18:25:52.073366: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2021-08-09 18:26:22.377801: F tensorflow/stream_executor/lib/statusor.cc:34] Attempting to fetch value instead of handling error Internal: failed initializing StreamExecutor for CUDA device ordinal 1: Internal: failed call to cuDevicePrimaryCtxRetain: CUDA_ERROR_LAUNCH_FAILED: unspecified launch failure
Aborted (core dumped)
```

Anaconda Package List:
```
_ipyw_jlab_nb_ext_conf    0.1.0                    py38_0  
_libgcc_mutex             0.1                        main  
_tflow_select             2.1.0                       gpu  
absl-py                   0.13.0           py38h06a4308_0  
alabaster                 0.7.12             pyhd3eb1b0_0  
anaconda                  2021.05                  py38_0  
anaconda-client           1.7.2                    py38_0  
anaconda-navigator        2.0.3                    py38_0  
anaconda-project          0.9.1              pyhd3eb1b0_1  
anyio                     2.2.0            py38h06a4308_1  
appdirs                   1.4.4                      py_0  
argh                      0.26.2                   py38_0  
argon2-cffi               20.1.0           py38h27cfd23_1  
asn1crypto                1.4.0                      py_0  
astor                     0.8.1            py38h06a4308_0  
astroid                   2.5              py38h06a4308_1  
astropy                   4.2.1            py38h27cfd23_1  
astunparse                1.6.3                      py_0  
async_generator           1.10               pyhd3eb1b0_0  
atomicwrites              1.4.0                      py_0  
attrs                     20.3.0             pyhd3eb1b0_0  
autopep8                  1.5.6              pyhd3eb1b0_0  
babel                     2.9.0              pyhd3eb1b0_0  
backcall                  0.2.0              pyhd3eb1b0_0  
backports                 1.0                pyhd3eb1b0_2  
backports.functools_lru_cache 1.6.4              pyhd3eb1b0_0  
backports.shutil_get_terminal_size 1.0.0              pyhd3eb1b0_3  
backports.tempfile        1.0                pyhd3eb1b0_1  
backports.weakref         1.0.post1                  py_1  
beautifulsoup4            4.9.3              pyha847dfd_0  
bitarray                  2.1.0            py38h27cfd23_1  
bkcharts                  0.2                      py38_0  
black                     19.10b0                    py_0  
blas                      1.0                         mkl  
bleach                    3.3.0              pyhd3eb1b0_0  
blinker                   1.4              py38h06a4308_0  
blosc                     1.21.0               h8c45485_0  
bokeh                     2.3.2            py38h06a4308_0  
boto                      2.49.0                   py38_0  
bottleneck                1.3.2            py38heb32a55_1  
brotlipy                  0.7.0           py38h27cfd23_1003  
bzip2                     1.0.8                h7b6447c_0  
c-ares                    1.17.1               h27cfd23_0  
ca-certificates           2021.4.13            h06a4308_1  
cachetools                4.2.2              pyhd3eb1b0_0  
cairo                     1.16.0               hf32fb01_1  
certifi                   2020.12.5        py38h06a4308_0  
cffi                      1.14.5           py38h261ae71_0  
chardet                   4.0.0           py38h06a4308_1003  
click                     7.1.2              pyhd3eb1b0_0  
cloudpickle               1.6.0                      py_0  
clyent                    1.2.2                    py38_1  
colorama                  0.4.4              pyhd3eb1b0_0  
conda                     4.10.3           py38h06a4308_0  
conda-build               3.21.4           py38h06a4308_0  
conda-content-trust       0.1.1              pyhd3eb1b0_0  
conda-env                 2.6.0                         1  
conda-package-handling    1.7.3            py38h27cfd23_1  
conda-repo-cli            1.0.4              pyhd3eb1b0_0  
conda-token               0.3.0              pyhd3eb1b0_0  
conda-verify              3.4.2                      py_1  
contextlib2               0.6.0.post1                py_0  
coverage                  5.5              py38h27cfd23_2  
cryptography              3.4.7            py38hd23ed53_0  
cudatoolkit               10.1.243             h6bb024c_0  
cudnn                     7.6.5                cuda10.1_0  
cupti                     10.1.168                      0  
curl                      7.71.1               hbc83047_1  
cycler                    0.10.0                   py38_0  
cython                    0.29.23          py38h2531618_0  
cytoolz                   0.11.0           py38h7b6447c_0  
dask                      2021.4.0           pyhd3eb1b0_0  
dask-core                 2021.4.0           pyhd3eb1b0_0  
dataclasses               0.8                pyh6d0b6a4_7  
dbus                      1.13.18              hb2f20db_0  
decorator                 5.0.6              pyhd3eb1b0_0  
defusedxml                0.7.1              pyhd3eb1b0_0  
diff-match-patch          20200713                   py_0  
distributed               2021.4.1         py38h06a4308_0  
docutils                  0.17.1           py38h06a4308_1  
entrypoints               0.3                      py38_0  
et_xmlfile                1.0.1                   py_1001  
expat                     2.3.0                h2531618_2  
fastcache                 1.1.0            py38h7b6447c_0  
filelock                  3.0.12             pyhd3eb1b0_1  
flake8                    3.9.0              pyhd3eb1b0_0  
flask                     1.1.2              pyhd3eb1b0_0  
fontconfig                2.13.1               h6c09931_0  
freetype                  2.10.4               h5ab3b9f_0  
fribidi                   1.0.10               h7b6447c_0  
fsspec                    0.9.0              pyhd3eb1b0_0  
future                    0.18.2                   py38_1  
gast                      0.4.0                      py_0  
get_terminal_size         1.0.0                haa9412d_0  
gevent                    21.1.2           py38h27cfd23_1  
glib                      2.68.1               h36276a3_0  
glob2                     0.7                pyhd3eb1b0_0  
gmp                       6.2.1                h2531618_2  
gmpy2                     2.0.8            py38hd5f6e3b_3  
google-auth               1.21.3                     py_0  
google-auth-oauthlib      0.4.4              pyhd3eb1b0_0  
google-pasta              0.2.0                      py_0  
gperftools                2.7                  h767d802_2    conda-forge
graphite2                 1.3.14               h23475e2_0  
greenlet                  1.0.0            py38h2531618_2  
grpcio                    1.36.1           py38h2157cd5_1  
gst-plugins-base          1.14.0               h8213a91_2  
gstreamer                 1.14.0               h28cd5cc_2  
h5py                      2.10.0           py38h7918eee_0  
harfbuzz                  2.8.0                h6f93f22_0  
hdf5                      1.10.4               hb1b8bf9_0  
heapdict                  1.0.1                      py_0  
html5lib                  1.1                        py_0  
huggingface_hub           0.0.8                      py_0    huggingface
icu                       58.2                 he6710b0_3  
idna                      2.10               pyhd3eb1b0_0  
imageio                   2.9.0              pyhd3eb1b0_0  
imagesize                 1.2.0              pyhd3eb1b0_0  
importlib-metadata        3.10.0           py38h06a4308_0  
importlib_metadata        3.10.0               hd3eb1b0_0  
iniconfig                 1.1.1              pyhd3eb1b0_0  
intel-openmp              2021.2.0           h06a4308_610  
intervaltree              3.1.0                      py_0  
ipykernel                 5.3.4            py38h5ca1d4c_0  
ipython                   7.22.0           py38hb070fc8_0  
ipython_genutils          0.2.0              pyhd3eb1b0_1  
ipywidgets                7.6.3              pyhd3eb1b0_1  
isort                     5.8.0              pyhd3eb1b0_0  
itsdangerous              1.1.0              pyhd3eb1b0_0  
jbig                      2.1                  hdba287a_0  
jdcal                     1.4.1                      py_0  
jedi                      0.17.2           py38h06a4308_1  
jeepney                   0.6.0              pyhd3eb1b0_0  
jinja2                    2.11.3             pyhd3eb1b0_0  
joblib                    1.0.1              pyhd3eb1b0_0  
jpeg                      9b                   h024ee3a_2  
json5                     0.9.5                      py_0  
jsonschema                3.2.0                      py_2  
jupyter                   1.0.0                    py38_7  
jupyter-packaging         0.7.12             pyhd3eb1b0_0  
jupyter_client            6.1.12             pyhd3eb1b0_0  
jupyter_console           6.4.0              pyhd3eb1b0_0  
jupyter_core              4.7.1            py38h06a4308_0  
jupyter_server            1.4.1            py38h06a4308_0  
jupyterlab                3.0.14             pyhd3eb1b0_1  
jupyterlab_pygments       0.1.2                      py_0  
jupyterlab_server         2.4.0              pyhd3eb1b0_0  
jupyterlab_widgets        1.0.0              pyhd3eb1b0_1  
keras-preprocessing       1.1.2              pyhd3eb1b0_0  
keyring                   22.3.0           py38h06a4308_0  
kiwisolver                1.3.1            py38h2531618_0  
krb5                      1.18.2               h173b8e3_0  
lazy-object-proxy         1.6.0            py38h27cfd23_0  
lcms2                     2.12                 h3be6417_0  
ld_impl_linux-64          2.33.1               h53a641e_7  
libarchive                3.4.2                h62408e4_0  
libcurl                   7.71.1               h20c2e04_1  
libedit                   3.1.20210216         h27cfd23_1  
libev                     4.33                 h7b6447c_0  
libffi                    3.3                  he6710b0_2  
libgcc-ng                 9.1.0                hdf63c60_0  
libgfortran-ng            7.3.0                hdf63c60_0  
liblief                   0.10.1               he6710b0_0  
libllvm10                 10.0.1               hbcb73fb_5  
libpng                    1.6.37               hbc83047_0  
libprotobuf               3.17.2               h4ff587b_1  
libsodium                 1.0.18               h7b6447c_0  
libspatialindex           1.9.3                h2531618_0  
libssh2                   1.9.0                h1ba5d50_1  
libstdcxx-ng              9.1.0                hdf63c60_0  
libtiff                   4.2.0                h85742a9_0  
libtool                   2.4.6             h7b6447c_1005  
libuuid                   1.0.3                h1bed415_2  
libuv                     1.40.0               h7b6447c_0  
libwebp-base              1.2.0                h27cfd23_0  
libxcb                    1.14                 h7b6447c_0  
libxml2                   2.9.10               hb55368b_3  
libxslt                   1.1.34               hc22bd24_0  
llvmlite                  0.36.0           py38h612dafd_4  
locket                    0.2.1            py38h06a4308_1  
lxml                      4.6.3            py38h9120a33_0  
lz4-c                     1.9.3                h2531618_0  
lzo                       2.10                 h7b6447c_2  
markdown                  3.3.4            py38h06a4308_0  
markupsafe                1.1.1            py38h7b6447c_0  
matplotlib                3.3.4            py38h06a4308_0  
matplotlib-base           3.3.4            py38h62a2d02_0  
mccabe                    0.6.1                    py38_1  
mistune                   0.8.4           py38h7b6447c_1000  
mkl                       2021.2.0           h06a4308_296  
mkl-service               2.3.0            py38h27cfd23_1  
mkl_fft                   1.3.0            py38h42c9631_2  
mkl_random                1.2.1            py38ha9443f7_2  
mock                      4.0.3              pyhd3eb1b0_0  
more-itertools            8.7.0              pyhd3eb1b0_0  
mpc                       1.1.0                h10f8cd9_1  
mpfr                      4.0.2                hb69a4c5_1  
mpmath                    1.2.1            py38h06a4308_0  
msgpack-python            1.0.2            py38hff7bd54_1  
multipledispatch          0.6.0                    py38_0  
mypy_extensions           0.4.3                    py38_0  
navigator-updater         0.2.1                    py38_0  
nbclassic                 0.2.6              pyhd3eb1b0_0  
nbclient                  0.5.3              pyhd3eb1b0_0  
nbconvert                 6.0.7                    py38_0  
nbformat                  5.1.3              pyhd3eb1b0_0  
ncurses                   6.2                  he6710b0_1  
nest-asyncio              1.5.1              pyhd3eb1b0_0  
networkx                  2.5                        py_0  
nltk                      3.6.1              pyhd3eb1b0_0  
nose                      1.3.7           pyhd3eb1b0_1006  
notebook                  6.3.0            py38h06a4308_0  
numba                     0.53.1           py38ha9443f7_0  
numexpr                   2.7.3            py38h22e1b3c_1  
numpy                     1.20.1           py38h93e21f0_0  
numpy-base                1.20.1           py38h7d8b39e_0  
numpydoc                  1.1.0              pyhd3eb1b0_1  
oauthlib                  3.1.1              pyhd3eb1b0_0  
olefile                   0.46                       py_0  
openpyxl                  3.0.7              pyhd3eb1b0_0  
openssl                   1.1.1k               h27cfd23_0  
opt_einsum                3.3.0              pyhd3eb1b0_1  
packaging                 20.9               pyhd3eb1b0_0  
pandas                    1.2.4            py38h2531618_0  
pandoc                    2.12                 h06a4308_0  
pandocfilters             1.4.3            py38h06a4308_1  
pango                     1.45.3               hd140c19_0  
parso                     0.7.0                      py_0  
partd                     1.2.0              pyhd3eb1b0_0  
patchelf                  0.12                 h2531618_1  
path                      15.1.2           py38h06a4308_0  
path.py                   12.5.0                        0  
pathlib2                  2.3.5            py38h06a4308_2  
pathspec                  0.7.0                      py_0  
patsy                     0.5.1                    py38_0  
pcre                      8.44                 he6710b0_0  
pep8                      1.7.1                    py38_0  
perl                      5.26.2               h14c3975_0  
pexpect                   4.8.0              pyhd3eb1b0_3  
pickleshare               0.7.5           pyhd3eb1b0_1003  
pillow                    8.2.0            py38he98fc37_0  
pip                       21.0.1           py38h06a4308_0  
pixman                    0.40.0               h7b6447c_0  
pkginfo                   1.7.0            py38h06a4308_0  
pluggy                    0.13.1           py38h06a4308_0  
ply                       3.11                     py38_0  
prometheus_client         0.10.1             pyhd3eb1b0_0  
prompt-toolkit            3.0.17             pyh06a4308_0  
prompt_toolkit            3.0.17               hd3eb1b0_0  
protobuf                  3.17.2           py38h295c915_0  
psutil                    5.8.0            py38h27cfd23_1  
ptyprocess                0.7.0              pyhd3eb1b0_2  
py                        1.10.0             pyhd3eb1b0_0  
py-lief                   0.10.1           py38h403a769_0  
pyasn1                    0.4.8                      py_0  
pyasn1-modules            0.2.8                      py_0  
pycodestyle               2.6.0              pyhd3eb1b0_0  
pycosat                   0.6.3            py38h7b6447c_1  
pycparser                 2.20                       py_2  
pycurl                    7.43.0.6         py38h1ba5d50_0  
pydocstyle                6.0.0              pyhd3eb1b0_0  
pyerfa                    1.7.3            py38h27cfd23_0  
pyflakes                  2.2.0              pyhd3eb1b0_0  
pygments                  2.8.1              pyhd3eb1b0_0  
pyjwt                     2.1.0            py38h06a4308_0  
pylint                    2.7.4            py38h06a4308_1  
pyls-black                0.4.6                hd3eb1b0_0  
pyls-spyder               0.3.2              pyhd3eb1b0_0  
pyodbc                    4.0.30           py38he6710b0_0  
pyopenssl                 20.0.1             pyhd3eb1b0_1  
pyparsing                 2.4.7              pyhd3eb1b0_0  
pyqt                      5.9.2            py38h05f1152_4  
pyrsistent                0.17.3           py38h7b6447c_0  
pysocks                   1.7.1            py38h06a4308_0  
pytables                  3.6.1            py38h9fd0a39_0  
pytest                    6.2.3            py38h06a4308_2  
python                    3.8.8                hdb3f193_5  
python-dateutil           2.8.1              pyhd3eb1b0_0  
python-flatbuffers        1.12               pyhd3eb1b0_0  
python-jsonrpc-server     0.4.0                      py_0  
python-language-server    0.36.2             pyhd3eb1b0_0  
python-libarchive-c       2.9                pyhd3eb1b0_1  
python_abi                3.8                      1_cp38    huggingface
pytz                      2021.1             pyhd3eb1b0_0  
pywavelets                1.1.1            py38h7b6447c_2  
pyxdg                     0.27               pyhd3eb1b0_0  
pyyaml                    5.4.1            py38h27cfd23_1  
pyzmq                     20.0.0           py38h2531618_1  
qdarkstyle                2.8.1                      py_0  
qt                        5.9.7                h5867ecd_1  
qtawesome                 1.0.2              pyhd3eb1b0_0  
qtconsole                 5.0.3              pyhd3eb1b0_0  
qtpy                      1.9.0                      py_0  
readline                  8.1                  h27cfd23_0  
regex                     2021.4.4         py38h27cfd23_0  
requests                  2.25.1             pyhd3eb1b0_0  
requests-oauthlib         1.3.0                      py_0  
ripgrep                   12.1.1                        0  
rope                      0.18.0                     py_0  
rsa                       4.7.2              pyhd3eb1b0_1  
rtree                     0.9.7            py38h06a4308_1  
ruamel_yaml               0.15.100         py38h27cfd23_0  
sacremoses                master                     py_0    huggingface
scikit-image              0.18.1           py38ha9443f7_0  
scikit-learn              0.24.1           py38ha9443f7_0  
scipy                     1.6.2            py38had2a1c9_1  
seaborn                   0.11.1             pyhd3eb1b0_0  
secretstorage             3.3.1            py38h06a4308_0  
send2trash                1.5.0              pyhd3eb1b0_1  
sentencepiece             0.1.91           py38hbf85e49_3    conda-forge
setuptools                52.0.0           py38h06a4308_0  
simplegeneric             0.8.1                    py38_2  
singledispatch            3.6.1           pyhd3eb1b0_1001  
sip                       4.19.13          py38he6710b0_0  
six                       1.15.0           py38h06a4308_0  
sniffio                   1.2.0            py38h06a4308_1  
snowballstemmer           2.1.0              pyhd3eb1b0_0  
sortedcollections         2.1.0              pyhd3eb1b0_0  
sortedcontainers          2.3.0              pyhd3eb1b0_0  
soupsieve                 2.2.1              pyhd3eb1b0_0  
sphinx                    4.0.1              pyhd3eb1b0_0  
sphinxcontrib             1.0                      py38_1  
sphinxcontrib-applehelp   1.0.2              pyhd3eb1b0_0  
sphinxcontrib-devhelp     1.0.2              pyhd3eb1b0_0  
sphinxcontrib-htmlhelp    1.0.3              pyhd3eb1b0_0  
sphinxcontrib-jsmath      1.0.1              pyhd3eb1b0_0  
sphinxcontrib-qthelp      1.0.3              pyhd3eb1b0_0  
sphinxcontrib-serializinghtml 1.1.4              pyhd3eb1b0_0  
sphinxcontrib-websupport  1.2.4                      py_0  
spyder                    4.2.5            py38h06a4308_0  
spyder-kernels            1.10.2           py38h06a4308_0  
sqlalchemy                1.4.15           py38h27cfd23_0  
sqlite                    3.35.4               hdfb4753_0  
statsmodels               0.12.2           py38h27cfd23_0  
sympy                     1.8              py38h06a4308_0  
tbb                       2020.3               hfd86e86_0  
tblib                     1.7.0                      py_0  
tensorboard               2.4.0              pyhc547734_0  
tensorboard-plugin-wit    1.6.0                      py_0  
tensorflow                2.4.1           gpu_py38h8a7d6ce_0  
tensorflow-base           2.4.1           gpu_py38h29c2da4_0  
tensorflow-estimator      2.4.1              pyheb71bc4_0  
tensorflow-gpu            2.4.1                h30adc30_0  
termcolor                 1.1.0            py38h06a4308_1  
terminado                 0.9.4            py38h06a4308_0  
testpath                  0.4.4              pyhd3eb1b0_0  
textdistance              4.2.1              pyhd3eb1b0_0  
threadpoolctl             2.1.0              pyh5ca1d4c_0  
three-merge               0.1.1              pyhd3eb1b0_0  
tifffile                  2020.10.1        py38hdd07704_2  
tk                        8.6.10               hbc83047_0  
tokenizers                0.10.3                   py38_0    huggingface
toml                      0.10.2             pyhd3eb1b0_0  
toolz                     0.11.1             pyhd3eb1b0_0  
tornado                   6.1              py38h27cfd23_0  
tqdm                      4.59.0             pyhd3eb1b0_1  
traitlets                 5.0.5              pyhd3eb1b0_0  
transformers              4.8.1                      py_0    huggingface
typed-ast                 1.4.2            py38h27cfd23_1  
typing_extensions         3.7.4.3            pyha847dfd_0  
ujson                     4.0.2            py38h2531618_0  
unicodecsv                0.14.1                   py38_0  
unixodbc                  2.3.9                h7b6447c_0  
urllib3                   1.26.4             pyhd3eb1b0_0  
watchdog                  1.0.2            py38h06a4308_1  
wcwidth                   0.2.5                      py_0  
webencodings              0.5.1                    py38_1  
werkzeug                  1.0.1              pyhd3eb1b0_0  
wheel                     0.36.2             pyhd3eb1b0_0  
widgetsnbextension        3.5.1                    py38_0  
wrapt                     1.12.1           py38h7b6447c_1  
wurlitzer                 2.1.0            py38h06a4308_0  
xlrd                      2.0.1              pyhd3eb1b0_0  
xlsxwriter                1.3.8              pyhd3eb1b0_0  
xlwt                      1.3.0                    py38_0  
xmltodict                 0.12.0                     py_0  
xz                        5.2.5                h7b6447c_0  
yaml                      0.2.5                h7b6447c_0  
yapf                      0.31.0             pyhd3eb1b0_0  
zeromq                    4.3.4                h2531618_0  
zict                      2.0.0              pyhd3eb1b0_0  
zipp                      3.4.1              pyhd3eb1b0_0  
zlib                      1.2.11               h7b6447c_3  
zope                      1.0                      py38_1  
zope.event                4.5.0                    py38_0  
zope.interface            5.3.0            py38h27cfd23_0  
zstd                      1.4.5                h9ceee32_0
```

**Comment by tilakrayal** [comment]
@Phill-at-GE ,

Can you please try installing TensorFlow v2.4 with CUDA 11.0 and cuDNN 8 and check if you are facing the same error. For more information please take a look at the [tested build configurations](https://www.tensorflow.org/install/source#gpu).Thanks!

**Comment by Phill-at-GE** [comment]
Thanks for the response! So, it's a little confusing with the way the Conda Packages are set up, along with the pre-installed NVIDIA driver and CUDA toolkit. 

Based on the link you sent, the compatibilities are: 
```
Version | Python version | Compiler | Build tools | cuDNN | CUDA
tensorflow-2.5.0 | 3.6-3.9 | GCC 7.3.1 | Bazel 3.7.2 | 8.1 | 11.2
tensorflow-2.4.0 | 3.6-3.8 | GCC 7.3.1 | Bazel 3.1.0 | 8.0 | 11.0
tensorflow-2.3.0 | 3.5-3.8 | GCC 7.3.1 | Bazel 3.1.0 | 7.6 | 10.1
```
Based on the Anaconda Packages, the following are available to my Linux system. 
```
cudnn                          7.6.5       cuda9.2_0  pkgs/main           
cudnn                          8.2.1      cuda11.3_0  pkgs/main 

cudatoolkit                 11.0.221      h6bb024c_0  pkgs/main           
cudatoolkit                   11.3.1      h2bc3f7f_2  pkgs/main
   
tensorflow                     2.4.1 gpu_py38h8a7d6ce_0  pkgs/main                
tensorflow                     2.4.1 mkl_py38hb2083e0_0  pkgs/main                 
tensorflow                     2.5.0 eigen_py38h94feaef_0  pkgs/main           
tensorflow                     2.5.0 mkl_py38hce4fbe1_0  pkgs/main           
```

I can't install the needed `cuDNN==8.0` for TF 2.4 (or even the `cuDNN==8.1` for TF 2.5). Also, I can't upgrade to `tensorflow-2.5.0` since conda doesn't provide a gpu version of TensorFlow for that (I know that TF 2.x on, that TF automatically searches for GPU's but I have found with Conda that you must install the gpu version in order to get GPU's to even be recognized). 

The pre-installed drivers/CUDA library on the Linux system is (nvidia-smi):
```
NVIDIA-SMI 460.32.03    Driver Version: 460.32.03    CUDA Version: 11.2 
```
So, maybe because of this, I can't even use TF 2.4 unless I uninstall the current CUDA Version: 11.2?

In summary, it seems like it is impossible to get the correct configuration required because of:
1) Anaconda packages don't support cuDNN (for either TF 2.4 or TF 2.5)
2) Anaconda packages don't support TF 2.5 with GPU support 
3) The installed CUDA version on my Linux is only for  TF 2.5.

**Comment by tilakrayal** [comment]
@Phill-at-GE ,

As mentioned, please try to install the tf v2.4 with compatible CUDA and cuDNN.Also installation issues within the Anaconda environment are tracked in the Anaconda repo.Please submit the new issue in Anaconda repo.Thanks!
